doc/upgrading.rst

```diff
@@ -0,0 +1,45 @@
+=========
+Upgrading
+=========
+
+This page describes how to upgrade the Nextstrain CLI--the ``nextstrain``
+command--itself, without also upgrading the entire Nextstrain
+:term:`docs:runtime`.
+
+The first step is to figure out what kind of Nextstrain CLI installation you
+have.
+
+If you run ``conda activate nextstrain`` before running ``nextstrain``
+commands, then you're using a **Conda-based installation** like we describe in
+the Nextstrain :doc:`installation steps <docs:install>`.  In this case, run the
+following to upgrade:
+
+.. code-block:: bash
+
+    conda activate nextstrain
+    mamba update -c conda-forge -c bioconda nextstrain-cli
+
+Otherwise, you're using another installation method and running:
+
+.. code-block:: console
+    :emphasize-lines: 6
+
+    $ nextstrain check-setup
+    A new version of nextstrain-cli, X.Y.Z, is available!  You're running A.B.C.
+
+    Upgrade by running:
+
+        [UPGRADE COMMAND]
```

Comment:
We should somehow add the `mamba update` instructions in this output, otherwise users following the central installation docs will see `pip install` which is not advised for conda-based installations.

Reply:
Yep, see https://github.com/nextstrain/cli/issues/114.

Reply:
Ah ok, missed your comment above.